man/systemd-udevd.service.xml

```diff
@@ -105,6 +105,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-s=</option></term>
+        <term><option>--timeout-signal=</option></term>
+        <listitem>
+          <para>Set the signal which systemd-udevd will send to the forked off
+          process after reaching event timeout.</para>
+        </listitem>
```

Comment:
please mention relationship to the option in the config file, i.e. which one overrides which (and maybe here too what the default signal is)

Reply:
this was not addressed?

Reply:
Fixed